fastlane_core/lib/fastlane_core/configuration/configuration.rb

```diff
@@ -200,7 +200,7 @@ def fetch(key, ask: true)
 
       while value.nil?
         UI.important("To not be asked about this value, you can specify it using '#{option.key}'")
-        value = ask("#{option.description}: ")
+        value = ask("#{option.description}: ") { |q| q.echo = !option.sensitive }
```

Comment:
I think the code is OK as written if we want the result to be that sensitive things don't have any echoed input.

I think your suggestion would make all non-sensitive params show '*' as their echoed input.

`q.echo = '*' if option.sensitive` would also be good, if we want sensitive things to show asterisks as the echoed input.

https://github.com/JEG2/highline#usage